beeai/common/models.py

```diff
@@ -30,3 +35,129 @@ class CVEEligibilityResult(BaseModel):
         default=None,
         description="Error message if the issue cannot be processed"
     )
+
+
+class TriageInputSchema(BaseModel):
+    """Input schema for the triage agent - metadata for a JIRA issue task."""
+    issue: str = Field(description="JIRA issue key (e.g., RHEL-12345)")
+    
+    def to_dict(self) -> Dict[str, Any]:
+        """Convert to dictionary for compatibility with existing code."""
```

Comment:
What existing code? This is just a rename of `model_dump()` to `to_dict()` (same as `to_json()` below), do we really need it?